creator-node/src/fileManager.js

```diff
@@ -288,12 +289,16 @@ async function saveFileForMultihashToFS (serviceRegistry, logger, multihash, exp
       throw new Error(`Error during content verification for multihash ${multihash} ${e.message}`)
     }
 
-    _printDecisionTreeObj(decisionTree, logger)
+    // If error, return boolean failure indicator + print logs
```

Comment:
this removes the decision tree log on success. i think this is actually good to reduce unnecessary logging, just confirming intent here

Reply:
yep exactly that was the intent